Packs/SimpleAPIProxy/Integrations/SimpleAPIProxy/SimpleAPIProxy_description.md

```diff
@@ -1,6 +1,6 @@
 #### Configuration
-- ***Lists to Publish*** - This is the comma separated list of list names you with to publish under the configured instance. Example: mylist1,mylist2,mylist3
-- ***List Items on Individual Lines*** - For XSOAR Lists that are comma separated text, this will take all the list items and return them on a separate line, one list item per line.
+- ***Lists to Publish*** - This is the comma-separated list of list names you with to publish under the configured instance. Example: mylist1,mylist2,mylist3
+- ***List Items on Individual Lines*** - For XSOAR Lists that are comma-separated text, this will take all the list items and return them on a separate line, one list item per line.
```

Comment:
```suggestion
- ***List Items on Individual Lines*** - For Cortex XSOAR Lists that are comma-separated text, this will take all the list items and return them on a separate line, one list item per line.
```